experiment/run_experiment.py

```diff
@@ -57,6 +58,9 @@
                                  r'^third_party/oss-fuzz/build/|'
                                  r'^docker/generated.mk$|'
                                  r'^docs/)')
+OSS_FUZZ_CORPUS_BACKUP_URL_FORMAT = (
```

Comment:
nit: does this need to be a global? if yes, does it need to be public (ie without the _ prefix)?

Reply:
Constants we usually try to put in global all over code. Added _ prefix.